### ISSUE TITLE
calamares-nixos-extensions: unfree checkbox cannot be tabbed out of

### ISSUE THREAD

**Issue by Pandapip1** [NixOS/nixpkgs#456067]
### Nixpkgs version

- Unstable (25.11)

### Describe the bug

I expect to be able to use only a keyboard for operating system installations. However, when you get to the unfree software checkbox, once that checkbox has tab focus, neither tab nor shift+tab will work anymore. This is also an accessibility issue.

### Steps to reproduce

1. Open nixos installer
2. Fill out as usual, using tab complete instead of a mouse
3. Get to the unfree software selection
4. Tab to the checkbox
5. Use space or enter (I forget) to tick it
6. You are now stuck and have to find a spare mouse.

### Expected behaviour

Tabbing should work

### Relevant log output

```console

```

### System metadata

N.A.

### Notify maintainers

@vlinkz

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)

### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+bug%22+-label%3A%226.topic%3A+darwin%22+-label%3A%226.topic%3A+nixos%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc
